data/avro-schema/src/main/resources/avro/net/corda/data/p2p/app/AuthenticatedMessageHeader.avsc

```diff
@@ -38,6 +38,16 @@
       "doc": "This value identifies the upstream user of the p2p layer that this message is sent from and should be received by. It can be used to filter incoming messages from the p2p layer and process only the ones destined for a specific system.",
       "name": "subsystem",
       "type": "string"
+    },
+    {
+      "doc": "Indicates what type of membership information should be used for P2P communication: pending, active, active or suspended. Should be only set by MGM services. Default value should be used everywhere else, which means that active member information will be used.",
```

Comment:
_"Should be only set by MGM services."_ Is that correct? Isn't it used by any service creating an authenticated message header? If I understand right, would a better description here mention that non-MGM services should only use `ACTIVE`, and only MGM services will use other options?

Reply:
Addressed in [9e55b51](https://github.com/corda/corda-api/pull/838/commits/9e55b51eb09a822c5ab8e20083c34edceb8bfa97)